docs/guides/cloud/gitlab-integration.mdx

```diff
@@ -75,6 +102,22 @@ from merging any MRs with flaky tests.
 
 You can manage this behavior on your project's **Project Settings** page.
 
+<DocsImage
+  src="/img/dashboard/gitlab-integration/gitlab-status-checks.jpg"
+  width={800}
+/>
+
+:::tip
```

Comment:
@pstakoun let me know what you think of these Flakey test CTAs and if you want any changes to them.